### PR TITLE
Update to a version of layoutlib that runs with JDK 11

### PR DESCRIPTION
**Pull request by jrodbx**
Android Studio now only runs on Java 11, so let's migrate this tool over!

This (still unpublished) version of layoutlib will point to b169460 which adds a new dedicated artifact for java 11.

```
b169460 Update JDK11 layoutlib jar following Bridge API changes by Jerome Gaillard · 1 year, 3 months ago
355d1bb Adds a native version of layoutlib for Linux by Jerome Gaillard · 1 year, 3 months ago
0b01c42 Update layoutlib prebuilt by Jerome Gaillard · 1 year, 3 months ago
7e60e9e Add a version of layoutlib that runs with JDK 11 by Jerome Gaillard · 1 year, 4 months ago
cad07a9 Add framework_res.jar to prebuilts by Sergey Prigogin · 1 year, 4 months ago
8e25155 Update layoutlib prebuilt by Jerome Gaillard · 1 year, 4 months ago
af3a2b4 Update layoutlib prebuilt by Fedor Kudasov · 1 year, 4 months ago
fca1605 Update layoutlib prebuilt by Fedor Kudasov · 1 year, 4 months ago
6bc38e5 Update layoutlib prebuilt by Jerome Gaillard · 1 year, 4 months ago
a206c6b Update layoutlib prebuilt by Fedor Kudasov · 1 year, 4 months ago
5017c17 Update layoutlib prebuilt by Fedor Kudasov · 1 year, 5 months ago
af48676 Update layoutlib prebuilt by Jerome Gaillard · 1 year, 5 months ago
9612c10 Update layoutlib prebuilt by Jerome Gaillard · 1 year, 5 months ago
48c9cfe Update layoutlib prebuilt to a version based on Android Q by Jerome Gaillard · 1 year, 5 months ago
47bb9b9 Update layoutlib prebuilt by Sergey Prigogin · 1 year, 5 months ago
121d1f6 Revert "Update layoutlib prebuilt" by Jerome Gaillard · 1 year, 5 months ago
83f512f Update layoutlib prebuilt by Fedor Kudasov · 1 year, 5 months ago
4d2e645 Restore fonts to previous version by Diego Perez · 1 year, 5 months ago
e73c086 Update layoutlib prebuilt by Fedor Kudasov · 1 year, 5 months ago
64efe11 Revert Change Icb63715f by Dana Dahlstrom · 1 year, 6 months ago
1a8ff9c Update layoutlib prebuilt to be based on Android Q by Jerome Gaillard · 1 year, 6 months ago
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -4,8 +4,8 @@ buildscript {
   ext.versions = [
     kotlin: '1.4.10',
     agp: '3.6.1',
-    layoutlib: '4.0.0-alpha01-1a8ff9c', // "should" be similar to versions.agp
-    androidTools: '26.5.0', // agp + 23.0.0 = androidTools
+    layoutlib: '4.0.0-alpha02-b169460', // "should" be similar to versions.agp
+    androidTools: '26.6.0', // agp + 23.0.0 = androidTools
```
Comment (jrodbx) [inline build.gradle:8]: bumps layoutlib-api to an updated version, due to a change in Bridge.init(...):

https://android.googlesource.com/platform/prebuilts/studio/layoutlib/+/0b01c42f2fa0ebc760f9c92ecd9cca12f8da5af4

https://android.googlesource.com/platform/frameworks/layoutlib/+/ec0913c9da850d6e98850d4a81a3742eabd97d92%5E%21/#F0

File: paparazzi/src/main/java/app/cash/paparazzi/internal/Renderer.kt [hunk 2]
```diff
@@ -74,6 +74,7 @@ internal class Renderer(
           DeviceConfig.loadProperties(buildProp),
           fontLocation,
           null,
+          null,
           DeviceConfig.getEnumMap(attrs),
           logger
```
Comment (jrodbx) [inline paparazzi/src/main/java/app/cash/paparazzi/internal/Renderer.kt:79]:
```
     public boolean init(
             Map<String,String> platformProperties,
             File fontLocation,
+            String nativeLibPath,
             String icuDataPath,
             Map<String, Map<String, Integer>> enumValueMap,
             LayoutLog log) {
```

File: paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt [hunk 3]
```diff
@@ -297,6 +297,7 @@ class Paparazzi(
     versionClass
         .getDeclaredField("SDK_INT")
         .apply {
+          isAccessible = true
```
Comment (gamepro65) [inline paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt:300]: Did this not work before?

**Review by gamepro65 — APPROVED** [review]

File: paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt (hunk 3 above)
Comment (jrodbx) [reply paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt:300]: worked on Java 8

**Pull request merged**